c++/include/orc/Vector.hh

```diff
@@ -122,6 +122,8 @@ namespace orc {
     DataBuffer<char*> data;
     // the length of each string
     DataBuffer<int64_t> length;
+    // string blob
+    DataBuffer<char> blob;
```

Comment:
How about make it private? Though we may need to add StringDirectColumnReader as a friend class.

Reply:
I would prefer to make it public since exposing the blob memory make it easier data conversion, and external caller would be able to access this memory via data array any way.

Reply:
OK, make sense